apps/src/code-studio/components/stageExtras/BonusLevels.jsx

```diff
@@ -15,6 +17,10 @@ const styles = {
     clear: 'both',
     overflow: 'hidden',
   },
+  solutionImage: {
+    border: '1px solid black',
+    marginBottom: 5,
+  }
```

Comment:
If this border is only on Artist & Play Lab thumbnails does that make them 2px taller than Maze thumbnails?